Detections/MultipleDataSources/ZincJan272021IOCs.yaml

```diff
@@ -122,6 +125,16 @@ query: |
   | where (CommandLine has "is64bitoperatingsystem" and CommandLine has "Debug\\Browse") or (CommandLine has_any (tokens))
   | project Type, TimeGenerated, Computer, Account, ProcessName, CommandLine 
   ),
+  ( WindowsEvent
```

Comment:
1. You can add 4688 here as well.
2. A query in a rule can be up to 10K bytes. Check that here. Also, since this query uses imDns, validation is skipped (a challenge with validation), so make sure you check the query, especially your part as union isfuzzy makes it easy to miss errors.